scene/property_utils.cpp

```diff
@@ -65,6 +65,17 @@ Variant PropertyUtils::get_property_default_value(const Object *p_object, const
 		*r_is_valid = false;
 	}
 
+	// Handle special case "Script" property
```

Comment:
```suggestion
	// Handle special case "Script" property.
```

Reply:
Just a question. Is this finalizing period a new formatting standard? I was following the comments around the code and almost none has a period at the end.

Reply:
No it's kind of old, but hasn't been enforced by fixing all offending comments so gradually updated

Reply:
Will keep in mind, thanks.